kolibri/core/exams/test/test_exam_api.py

```diff
@@ -332,3 +332,13 @@ def test_exam_with_valid_question_sources_succeeds(self):
         )
         response = self.post_new_exam(exam)
         self.assertEqual(response.status_code, 201)
+
+    def test_can_get_quiz_size(self):
+        self.login_as_admin()
+        response = self.client.get(
+            reverse("kolibri:core:exam-size"),
+            {
+                "class_id": self.classroom.id,
+            },
+        )
+        self.assertEqual(response.status_code, 200)
```

Comment:
For this and the lesson size tests, we should file a follow up issue to test that the response actually contains the correct file sizes.